src/GitVersion.Core/VersionCalculation/NextVersion.cs

```diff
@@ -0,0 +1,30 @@
+using GitVersion.Extensions;
+using GitVersion.Model.Configuration;
+
+namespace GitVersion.VersionCalculation;
+
+public class NextVersion
```

Comment:
I like how you encapsulate everything related to the "next version" into a class. 👍🏼

Reply:
Thank you very much for the positive comments.